core/services/relay/evm/mercury/persistence_manager.go

```diff
@@ -37,10 +38,11 @@ type PersistenceManager struct {
 	pruneFrequency        time.Duration
 }
 
-func NewPersistenceManager(lggr logger.Logger, orm ORM, jobID int32, maxTransmitQueueSize int, flushDeletesFrequency, pruneFrequency time.Duration) *PersistenceManager {
+func NewPersistenceManager(lggr logger.Logger, serverURL string, orm ORM, jobID int32, maxTransmitQueueSize int, flushDeletesFrequency, pruneFrequency time.Duration) *PersistenceManager {
```

Comment:
are persistence manager completely independent? if a mercury reporting plugin writes to two servers and one of those refuses some data or otherwise fails, does the reporting plugin care? asked differently are there any implicit or explicit assumptions when reading from the persistence manager that the target servers are in sync or otherwise have coupled contraints?

Reply:
Yes they are independent, each PersistenceManager is scoped to a unique URL

Reply:
> if a mercury reporting plugin writes to two servers and one of those refuses some data or otherwise fails, does the reporting plugin care?

Nope, reporting plugin doesn't know or care.

> are there any implicit or explicit assumptions when reading from the persistence manager that the target servers are in sync or otherwise have coupled contraints

Nope the queues operate completely independently, by design.